w5500-lwIP.h

```diff
@@ -13,15 +13,15 @@ class Wiznet5500lwIP: public Wiznet5500 {
 
     // start with dhcp client
     // default mac-address is inferred(+modified) from esp8266's STA one
-    boolean begin (const uint8_t *macAddress = NULL, uint16_t mtu = 1500);
+    boolean begin(SPIClass &spi, const uint8_t *macAddress = NULL, uint16_t mtu = 1500);
     
     // to be called regularly
     err_t loop ();
 
     const netif* getNetIf   () const { return &_netif; }
-    IPAddress    localIP    () const { return IPAddress(_netif.ip_addr.addr); }
-    IPAddress    subnetMask () const { return IPAddress(_netif.netmask.addr); }
-    IPAddress    gatewayIP  () const { return IPAddress(_netif.gw.addr); }
+    IPAddress    localIP    () const { return IPAddress(_netif.ip_addr.u_addr.ip4.addr); }
```

Comment:
does esp32 support IPv6 (then is using lwIP-v2)?
if that's so then there are lwIP macro to abstract the IPv4 address
I think it is `ip4_addr_get_u32(&_netif.{ip_addr,netmask,gw})` (see `<lwip>/include/ip4_addr.h`)